<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstNestedTable</class>
 <widget class="QMainWindow" name="firstNestedTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>firstNestedTable</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnect">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="accept">
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="New_Database"/>
    <addaction name="actionOpen_Database"/>
   </widget>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="New_Database"/>
   <addaction name="actionOpen_Database"/>
   <addaction name="action_AddItem"/>
   <addaction name="action_RemoveItem"/>
   <addaction name="actionClose"/>
   <addaction name="actionAuto_Populate"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="qrc/res.qrc">
     <normaloff>:/close2.png</normaloff>:/close2.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="New_Database">
   <property name="icon">
    <iconset resource="qrc/res.qrc">
     <normaloff>:/new_database.png</normaloff>:/new_database.png</iconset>
   </property>
   <property name="text">
    <string>New Database</string>
   </property>
   <property name="toolTip">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="icon">
    <iconset resource="qrc/res.qrc">
     <normaloff>:/database.png</normaloff>:/database.png</iconset>
   </property>
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="action_AddItem">
   <property name="icon">
    <iconset resource="qrc/res.qrc">
     <normaloff>:/working.png</normaloff>:/working.png</iconset>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </action>
  <action name="action_RemoveItem">
   <property name="icon">
    <iconset resource="qrc/res.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="text">
    <string>Remove Item</string>
   </property>
  </action>
  <action name="actionAuto_Populate">
   <property name="icon">
    <iconset resource="qrc/res.qrc">
     <normaloff>:/cloud_connection.png</normaloff>:/cloud_connection.png</iconset>
   </property>
   <property name="text">
    <string>Auto-Populate</string>
   </property>
   <property name="toolTip">
    <string>Load from image directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc/res.qrc"/>
 </resources>
 <connections/>
</ui>
